nixos/modules/services/home-automation/matter-server.nix

```diff
@@ -0,0 +1,123 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.matter-server;
+in
+
+{
+  meta.maintainers = with lib.maintainers; [ hexa ];
+
+  options.services.matter-server = with types; {
+    enable = mkEnableOption (lib.mdDoc "Matter-server");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.python-matter-server;
+      defaultText = literalExpression "pkgs.python-matter-server";
+      description = "Package containing the matter-server executable.";
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 5580;
+      defaultText = "5580";
```

Comment:
```suggestion
```

Only required when the default value is a complex expression or references other values.

Reply:
Thanks for the suggestion! TIL.